README.md

```diff
@@ -588,6 +607,10 @@ To manually define the permissions policy of your Zappa execution role, you must
 
 Ongoing discussion about the minimum policy requirements necessary for a Zappa deployment [can be found here](https://github.com/Miserlou/Zappa/issues/244).
 
+#### Using API Gateway Authorizer
+
```

Comment:
Is a commit missing? I'm confused. If I look at your doc, there is just a random section header that says `Using API Gateway Authorizer` but there is nothing below it, just the next section?